crates/oxide/src/extractor/mod.rs

```diff
@@ -139,6 +140,41 @@ impl<'a> Extractor<'a> {
 
         extracted
     }
+
+    pub fn extract_css_variables_from_css_files(&mut self) -> Vec<Extracted<'a>> {
```

Comment:
Bit of a mouthful, open to suggestions. Note: this is completely internal API.

Reply:
`extract_variables_from_css` if you want a shorter name but this is fine imo